payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -20,7 +20,9 @@ import org.json.JSONObject
  * See [PaymentMethodCreateParams] for PaymentMethod creation
  */
 @Parcelize
-data class PaymentMethod internal constructor(
+data class PaymentMethod
```

Comment:
Most of this is opened up due to PaymentOptionsAdapter.SavedPaymentMethod.getDescription